downstream/titles/dev-guide/master.adoc

```diff
@@ -7,6 +7,19 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= {PlatformName} Developer Guide
+= {PlatformName} Creator Guide
 
+
+
+== Preface
```

Comment:
This creates a Preface chapter consisting of one paragraph. Is this OK, or is it just a consequence of building an individual guide?